public/app/features/plugins/admin/components/InstallControls/InstallControlsButton.tsx

```diff
@@ -65,13 +66,19 @@ export function InstallControlsButton({
   const onInstall = async () => {
     trackPluginInstalled(trackingProps);
     const result = await install(plugin.id, latestCompatibleVersion?.version);
-    // refresh the store to have the new installed plugin
-    await fetchDetails(plugin.id);
     if (!errorInstalling && !('error' in result)) {
-      appEvents.emit(AppEvents.alertSuccess, [`Installed ${plugin.name}`]);
+      let successMessage = `Installed ${plugin.name}`;
+      if (config.pluginAdminExternalManageEnabled && configCore.featureToggles.managedPluginsInstall) {
+        successMessage =
+          'Plugin marked for installation. Please note that it can take from 30 s to 5 min for the plugin to be available for usage.';
```

Comment:
When we took this to vote in slack the winner was `Install requested, this may take a few minutes.` 🙂

Reply:
@sympatheticmoose I've updated the message and the video, please take a look again!